spark/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+# Inspired from official spark chart
```

Comment:
One thing on licensing - AFAIK original chart was Apache 2.0 licensed (which is compatible with GPLv3 in this direction), so may be a link to original one https://github.com/kubernetes/examples/tree/master/staging/spark and a bit more substantial attributions in README might be in order?

Of course that depends on how much inspiration was taken :)
Anyway, it's not copyright law, but an etiquette here.